src/observers/OpenURLObserver.ts

```diff
@@ -1,23 +0,0 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-import { vscode } from "../vscodeAdapter";
-import { BaseEvent, OpenURL } from "../omnisharp/loggingEvents";
-import { EventType } from "../omnisharp/EventType";
-
-export class OpenURLObserver {
-
-    constructor(private vscode: vscode) {
-    }
-
-    public post = (event: BaseEvent) => {
```

Comment:
this seemed entirely unnecessary - an event + observer just for a single location to open a URL